.github/workflows/robot/internal/bot/assign.go

```diff
@@ -155,8 +155,26 @@ func (b *Bot) findOriginal(ctx context.Context, organization string, repository
 	return n, nil
 }
 
+func dedup(author string, reviewers []string) []string {
+	m := map[string]bool{}
+
+	for _, reviewer := range reviewers {
+		if reviewer == author {
+			continue
+		}
+		m[reviewer] = true
+	}
+
+	var filtered []string
+	for k := range m {
+		filtered = append(filtered, k)
+	}
+
+	return filtered
+}
+
 func isBackport(unsafeBase string) bool {
 	return strings.HasPrefix(unsafeBase, "branch/v")
 }
 
-var pattern = regexp.MustCompile(`#([0-9]+)`)
+var pattern = regexp.MustCompile(`(https:\/\/github\.com\/gravitational\/teleport\/pull\/|#)([0-9]+)`)
```

Comment:
Why the change here?

Reply:
@espadolini Found an example where where the bot failed to detect a backport because the full PR URL was included instead of just the number: https://github.com/gravitational/teleport/pull/10944

Reply:
Is `gravitational/teleport` the only repo where the bot runs?

Reply:
```suggestion
var pattern = regexp.MustCompile(`(?:https:\/\/github\.com\/gravitational\/teleport\/pull\/|#)([0-9]+)`)
```
and then we don't have to shift up the match numbers?

Reply:
Right now yes, but in the future we will probably pull it out into it's own action. I can clean this up but that will make it a runtime error.